plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlCaseInsensitiveMapping.java

```diff
@@ -54,131 +50,20 @@ public final void destroy()
-                        AutoCloseable ignore3 = withTable(schemaName + ".some_table_name", "(c varchar(5))")) {
-                    assertThat(computeActual("SHOW SCHEMAS").getOnlyColumn()).contains("casesensitivename");
-                    assertThat(computeActual("SHOW SCHEMAS").getOnlyColumn().filter("casesensitivename"::equals)).hasSize(1); // TODO change io.trino.plugin.jdbc.JdbcClient.getSchemaNames to return a List
-                    assertQueryFails("SHOW TABLES FROM casesensitivename", "Failed to find remote schema name: Ambiguous name: casesensitivename");
-                    assertQueryFails("SELECT * FROM casesensitivename.some_table_name", "Failed to find remote schema name: Ambiguous name: casesensitivename");
-                }
-            }
-        }
-    }
-
-    @Test
-    public void testTableNameClash()
-            throws Exception
-    {
-        String[] nameVariants = {"casesensitivename", "CaseSensitiveName", "CASESENSITIVENAME"};
-        assertThat(Stream.of(nameVariants)
-                .map(name -> name.toLowerCase(ENGLISH))
-                .collect(toImmutableSet()))
-                .hasSize(1);
-
-        for (int i = 0; i < nameVariants.length; i++) {
-            for (int j = i + 1; j < nameVariants.length; j++) {
-                try (AutoCloseable ignore1 = withTable("tpch." + nameVariants[i], "(c varchar(5))");
-                        AutoCloseable ignore2 = withTable("tpch." + nameVariants[j], "(d varchar(5))")) {
-                    assertThat(computeActual("SHOW TABLES").getOnlyColumn()).contains("casesensitivename");
-                    assertThat(computeActual("SHOW TABLES").getOnlyColumn().filter("casesensitivename"::equals)).hasSize(1); // TODO, should be 2
-                    assertQueryFails("SHOW COLUMNS FROM casesensitivename", "Failed to find remote table name: Ambiguous name: casesensitivename");
-                    assertQueryFails("SELECT * FROM casesensitivename", "Failed to find remote table name: Ambiguous name: casesensitivename");
-                }
-            }
-        }
-    }
-
-    private AutoCloseable withSchema(String schemaName)
+    private void execute(String sql)
     {
-        execute(format("CREATE SCHEMA `%s`", schemaName));
-        return () -> execute(format("DROP SCHEMA `%s`", schemaName));
+        memSqlServer.execute(sql);
     }
 
-    /**
-     * @deprecated Use {@link TestTable} instead.
-     */
-    @Deprecated
-    private AutoCloseable withTable(String tableName, String tableDefinition)
+    @Override
+    protected Path getMappingFile()
     {
-        execute(format("CREATE TABLE %s %s", tableName, tableDefinition));
-        return () -> execute(format("DROP TABLE %s", tableName));
+        return mappingFile;
     }
 
-    private void execute(String sql)
+    @Override
+    protected SqlExecutor onRemoteDatabase()
     {
-        memSqlServer.execute(sql);
+        return memSqlServer::execute;
```

Comment:
You can inline the function here. So something like `return sql -> memSqlServer.execute(sql)` and then remove the `execute` function since it's not used anywhere anymore.

Reply:
ok, do the inline at onRemoteDataBase() of TestOracleCaseInsensitiveMapping.java too?